react-app/src/components/estate/EstatePage.js

```diff
@@ -1,46 +1,52 @@
-import React, { useEffect } from "react";
+import React, { useEffect, useState } from "react";
 import { useSelector, useDispatch } from "react-redux";
 import { useParams } from "react-router-dom";
 import './estatePage.css';
 import ImageDisplay from "./ImageDisplay";
 import DescriptionAvailCharter from "./DescriptionAvailCharter";
 import CritiqueDisplay from "./CritiqueDisplay";
 import { genEstates } from "../../store/estate";
+// import { genUsers } from "../../store/user";
 import LocationDisplay from "./LocationDisplay";
 
 
-// const estate = {
-// 	address: "10922 Corbly Gulch Rd, Belgrade, MT 59714",
-// 	owner_id: 2,
-// 	title: "Bozeman Overlook",
-// 	nightly_rate: 790,
-// 	type_id: 3,
-// 	description:
-// 		"Don’t miss the opportunity to stay at this unique cabin getaway! This private and cozy home books up fast with the unparalleled 360 views and outstanding accommodations on 100 acres featuring premium bedding, extensive new renovations, and comfortable entertainment space. The home is perfect for an intimate getaway or group gathering! Ideal location for access to trails minutes away, 15 mins to the outskirts of Bozeman, and surrounded by local event venues.",
-// };
+
 
 const EstatePage = () => {
 	const { id } = useParams();
 
 	const dispatch = useDispatch();
     
     useEffect(() => {
-        dispatch(genEstates());        
+        dispatch(genEstates());       
+		// dispatch(genUsers());
     }, [dispatch])
-    
-       
-    const estate = useSelector((state) => state.estates[id]);
-    console.log(estate);
 
+	const [users, setUsers] = useState([]);
 
-	// once state is constructed
-	// const estate = useSelector((state) => state.estate[id])
+	useEffect(() => {
```

Comment:
Can we remove this and the passed props and we can address the username display with the critiques object in an upcoming PR.